content/decadal/3_experiments.md

```diff
@@ -0,0 +1,33 @@
+---
+tag: Experiments
+title: Experiments contributing to the Decadal Climate Prediction Project
+img: calibration.png
+caption: An example showing the steps of the full calibration methods and calibration of the dynamically decomposed variables (C.H.O’Reilly et al.)
```

Comment:
Can the caption source be a link (or at least a DOI so I can find it)?

Reply:
I put the doi there now.